src/components/SideEffects.js

```diff
@@ -36,8 +36,20 @@ const SideEffects = observer(({ assembly }) => (
             </Time>
 
             { sr.reported_symptoms.length !== 0
-            ? <Info>{sr.reported_symptoms.join(", ")}</Info>
-            : <Info>{assembly.translate("progress.no_side_effects")}</Info>
+
+            ? <Info>
+                { sr
+                  .reported_symptoms
+                  .map(symptom_key => assembly.translate(`survey.symptoms.${symptom_key}`))
+                  .join(", ")}
+
+                { sr.nausea_rating }
```

Comment:
There are translations for these as well, at `"survey.symptoms.nausea_ratings.0"`, etc.